### PR TITLE
Radio button is now replaced by Icon

### PR DESCRIPTION
**Pull request by thatguynischal**
I have replaced radio button with an icon of dollar.

### CODE REVIEW

File: src/modules/projects/components/ProjectList.vue [hunk 1]
```diff
@@ -91,7 +92,7 @@ function search(a) {
           <td>{{ index + 1 }}</td>
           <td>{{ project.project_name }}</td>
           <td>{{ project.project_color_code }}</td>
-          <td>{{ project.client_id }}</td>
+          <td>{{ project.client.id }}</td>
```
[the remark this answers is not in the document]
Comment (thatguynischal) [reply src/modules/projects/components/ProjectList.vue:95]: no bro....it is now changed to project.client.id
image

[the remark this answers is not in the document]
Comment (thatguynischal) [reply src/modules/projects/components/ProjectList.vue:95]: nirmal bro ko task paxadi tah it'll change to project.client.name

[the remark this answers is not in the document]
Comment (shrestharikesh) [reply src/modules/projects/components/ProjectList.vue:95]: There is a conflict please make this branch up to date with main

**Review by shrestharikesh — APPROVED** [review]

**Review by shrestharikesh — APPROVED** [review]
looks good

**Pull request merged**